### PR TITLE
Increase coverage

### PR DESCRIPTION
**Pull request by sroy3**
A drop in the sea, but some quick wins to increase the coverage. Will try to increase more in upcoming PRs.

### CODE REVIEW

File: webview/setup-tests.js
Comment (sroy3) [inline webview/setup-tests.js:1]: Took me way too long to realize why `getThemeValue` wasn't called unless I changed its name.

File: webview/jest.config.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ module.exports = {
   collectCoverageFrom: [
     'src/**/*.{ts,tsx}',
     '!**/*.test.*',
-    '!src/stories/*',
+    '!src/stories/**/*.{ts,tsx}',
```
Comment (sroy3) [inline webview/jest.config.js:7]: Correctly removes the `IconWrapper` and `IconWrappers` components from the coverage.

**Comment by mattseddon** [comment]
We can drop the overall coverage requirement by a few percentage points but we need to get back on top of this.

**Comment by mattseddon** [comment]
I guess if we are aiming for 95% the pass/fail for coverage should be 95% on a PR. Need to discuss what's meaningful this week in the retro.

**Comment by sroy3** [comment]
> I guess if we are aiming for 95% the pass/fail for coverage should be 95% on a PR. Need to discuss what's meaningful this week in the retro.

That make sense.

**Review by mattseddon — APPROVED** [review]

**Pull request merged**